src/components/LoadingMessage/LoadingMessage.tsx

```diff
@@ -40,23 +40,22 @@ LoadingMessageTitle.propTypes = {
 	children: any,
 };
 
+/** Loading Message Body */
 export interface ILoadingMessageBodyProps extends StandardProps {
 	description?: string;
 }
 const LoadingMessageBody = (_props: ILoadingMessageBodyProps): null => null;
 LoadingMessageBody.displayName = 'LoadingMessage.Body';
 LoadingMessageBody.peek = {
-	description: `
-		Renders an \`<span>\` that represents the body of the
-		\`LoadingMessage\`.
-	`,
+	description: `Renders an \`<span>\` that represents the body of the \`LoadingMessage\`.`,
```

Comment:
nit:  It should be 
description: `Renders a

Reply:
👍